types/moko/LW001-BG-PRO/bluetooth.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://akenza.io/moko/lw001-bg-pro/bluetooth.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "bluetooth",
+  "title": "Bluetooth",
+  "properties": {
+    "macData": {
+      "title": "Wifi mac data",
+      "type": "array",
+      "description": "Wifi mac data",
+      "hideFromKpis": true
+    },
+    "timestamp": {
```

Comment:
is there a need to additionally output the timestamp or can it be the one of the sample?